tests/e2e/ctl_v3_snapshot_test.go

```diff
@@ -447,5 +447,6 @@ func hasKVs(t *testing.T, ctl *e2e.EtcdctlV3, kvs []testutils.KV, currentRev int
 		require.Equal(t, int64(baseRev+i), v.Kvs[0].CreateRevision)
 		require.Equal(t, int64(baseRev+i), v.Kvs[0].ModRevision)
 		require.Equal(t, int64(1), v.Kvs[0].Version)
+		require.True(t, int64(currentRev) >= v.Kvs[0].ModRevision)
```

Comment:
why add this in this PR although it isn't a big problem?

Reply:
The `currentRev` as argument is not used by function. The lint blames this.